### PR TITLE
ov2.0: modify python non model API demos [Part 1]

### PR DESCRIPTION
**Pull request by anzhella-pankratova**
Modify the following python demos with new OpenVINO 2.0 API:
- 3d_segmentation_demo
- face_detection_mtcnn_demo
- face_recognition_demo
- gesture_recognition_demo
- handwritten_text_recognition_demo
- human_pose_estimation_3d_demo

### CODE REVIEW

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

File: demos/face_detection_mtcnn_demo/python/face_detection_mtcnn_demo.py [hunk 1]
```diff
@@ -218,7 +227,8 @@ def main():
                 crop_img = preprocess_image(crop_img, 24, 24)
                 rnet_input.extend(crop_img)
 
-            rnet_res = exec_rnet.infer(inputs={rnet_input_blob: rnet_input})
+            infer_request_rnet.infer(inputs={rnet_input_tensor_name: rnet_input})
+            rnet_res = {name: infer_request_rnet.get_tensor(name).data for name in {rnet_roi_name, rnet_cls_name}}
```
Comment (eaidova) [inline demos/face_detection_mtcnn_demo/python/face_detection_mtcnn_demo.py:231]:
```suggestion
            rnet_res = {name: infer_request_rnet.get_tensor(name).data[:] for name in {rnet_roi_name, rnet_cls_name}}
```

File: demos/face_detection_mtcnn_demo/python/face_detection_mtcnn_demo.py [hunk 2]
```diff
@@ -239,7 +250,8 @@ def main():
                 crop_img = preprocess_image(crop_img, 48, 48)
                 onet_input.extend(crop_img)
 
-            onet_res = exec_onet.infer(inputs={onet_input_blob: onet_input})
+            infer_request_onet.infer(inputs={onet_input_tensor_name: onet_input})
+            onet_res = {name: infer_request_onet.get_tensor(name).data for name in {onet_roi_name, onet_cls_name, onet_pts_name}}
```
Comment (eaidova) [inline demos/face_detection_mtcnn_demo/python/face_detection_mtcnn_demo.py:254]:
```suggestion
            onet_res = {name: infer_request_onet.get_tensor(name).data[:] for name in {onet_roi_name, onet_cls_name, onet_pts_name}}
```

File: demos/handwritten_text_recognition_demo/python/handwritten_text_recognition_demo.py [hunk 3]
```diff
@@ -77,44 +76,50 @@ def main():
...
     log.info('The model {} is loaded to {}'.format(args.model, args.device))
 
     # Start sync inference
     start_time = perf_counter()
-    for i in range(args.number_iter):
-        preds = exec_net.infer(inputs={input_blob: input_image})
-        preds = preds[out_blob]
+    for _ in range(args.number_iter):
+        infer_request.infer(inputs={input_tensor_name: input_image})
+        preds = infer_request.get_tensor(output_tensor_name).data
```
Comment (eaidova) [inline demos/handwritten_text_recognition_demo/python/handwritten_text_recognition_demo.py:122]:
```suggestion
        preds = infer_request.get_tensor(output_tensor_name).data[:]
```

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by anzhella-pankratova** [comment]
Jenkins please retry a build

**Review by vladimir-dudnik — APPROVED** [review]

**Pull request merged**